examples/text-to-speech/hello.yaml

```diff
@@ -0,0 +1,8 @@
+version: 1
+prompt:
+  content: "Good Morning, Warsaw!"
+services:
+  models:
+    - 'elevenlabs/tts/eleven_monolingual_v1'
+  options:
```

Comment:
I'd mention somewhere that you can also use your own voice id (the hash) for voice